OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionTopicsFragment.java

```diff
@@ -180,6 +180,10 @@ private void showCourseDiscussionTopic() {
                     getArguments().getString(Router.EXTRA_DISCUSSION_TOPIC_ID),
                     getArguments().getString(Router.EXTRA_DISCUSSION_THREAD_ID),
                     courseData);
+
+            // Put discussion_topic_id to null, so that upon recreation of the fragment deep link
```

Comment:
// Setting this to null, so that upon recreation of the fragment, relevant activity
// shouldn't be auto-created again (e.g. due to a deep link).